aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/__main__.py

```diff
@@ -0,0 +1,173 @@
+"""An AWS fargate with crosswalk vpc"""
+
+import pulumi
+from pulumi import export, ResourceOptions, Config, StackReference, get_stack, get_project
+import json
+import pulumi_aws as aws
+
+#https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/#inter-stack-dependencies
+
+# read local config settings - network
+config = Config()
+
+# reading in vpc StackReference Path from local config
+mystackpath = config.get("mystackpath")
+
+# setting the StackReference
+mycrosswalkvpc = StackReference(f"{mystackpath}")
```

Comment:
f string shouldn't be needed here

```suggestion
mycrosswalkvpc = StackReference(mystackpath)
```

Reply:
Fixed